generative_data_prep/processors/article_tokenizer.py

```diff
@@ -288,6 +293,8 @@ def tokenize(self, completion: str,
         token_type_ids: List[int] = []
 
         if prompt:
+            if prompter_keyword or assistant_keyword:
```

Comment:
Would this fail if prompter_keyword is None but assistant_keyword is not None? Because you try to concat None with a string?